Sources/Mesosphere/RValue.swift

```diff
@@ -53,7 +53,11 @@ extension GIRGenFunction {
       argVals.reserveCapacity(args.count)
       for arg in args {
         let (newParent, value) = self.emitRValue(lastParent, arg)
-        argVals.append(value.copy(self).forward(self))
+        if value.value is Parameter {
```

Comment:
I'm not 100% on this definition but it seems to work in practice. We don't necessarily want the basic blocks downstream of the chuzzling to destroy the value they just switched on - the switch_constr block still owns them and they should be destroyed before an apply of the return continuation.